orm/internal/codegen/index.go

```diff
@@ -5,19 +5,18 @@ import (
 	"strings"
 
 	"github.com/iancoleman/strcase"
-	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 func (t tableGen) genIndexKeys() {
-	// interface that all keys must adhere to
+	// Interface that all keys must adhere to.
 	t.P("type ", t.indexKeyInterfaceName(), " interface {")
 	t.P("id() uint32")
-	t.P("values() []interface{}")
+	t.P("values() []any")
```

Comment:
Please revert. This changes how the code is generated. This enforces a minimum go version while it didn't before. The code generator should be as compatible as possible.